rainforest/resources.go

```diff
@@ -36,7 +36,7 @@ func (c *Client) getPaginatedResource(endpoint string, coll interface{}, collect
 
 	totalPagesHeader := res.Header.Get("X-Total-Pages")
 	if totalPagesHeader == "" {
-		return fmt.Errorf("Missing X-Total-Pages header in HTTP Response")
+		totalPagesHeader = "1"
```

Comment:
Do the newly paginated endpoints not have a total pages header? Or why this change? It seems fine, just curious.

Reply:
See response to @jbarber above, they're not paginated _yet_. Once they are, they'll have pagination headers like all the other paginated endpoints.